tests/frame/std_test.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any
+
+import narwhals.stable.v1 as nw
+from tests.utils import compare_dicts
+
+data = {"a": [1, 3, 2], "b": [4, 4, 6], "z": [7.0, 8, 9]}
+
+
+def test_std(constructor_eager: Any) -> None:
+    df = nw.from_native(constructor_eager(data))
```

Comment:
while we're changing this, can we use `constructor` instead of `constructor` eager? this test should work for lazy too